wetterdienst/cli.py

```diff
@@ -61,16 +65,16 @@ def run():
     Examples requesting stations:
 
       # Get list of all stations for daily climate summary data in JSON format
-      wetterdienst dwd stations --parameter=kl --resolution=daily --period=recent
+      wetterdienst dwd sites --parameter=kl --resolution=daily --period=recent
```

Comment:
Thanks for using `sites` here on the user interface level already. We also might want to adjust internally as well as on the `--station` parameter.